configFileProductions/spring2024ConfigFiles/nunuH_Hxx/LCFIJetEnabled/whizardddsimgaudi.conf

```
@@ -0,0 +1,43 @@
+[whizard2]
+Version = 2.8.3
+
+[ddsim]
+SteeringFile = cld_steer.py
+ExtraCLIArguments = --vertexSigma 0.0098 2.54e-5 0.646 0.0063 --physics.rejectPDGs 1,2,3,4,5,6,12,14,16,21,23,24,25
+
+[gaudiapp]
+ExecutableName = k4run
+SteeringFile = CLDReconstruction.py
+InputFileFlag = --inputFiles
+OutputFileFlag = --Output_REC.filename
+Arguments = --enableLCFIJet
```

```suggestion
Arguments = --enableLCFIJet --cms 240
```
Let's be explicit and not rely on the default for when there is another energy to be used.

---

Done, but only after submitting :(